pkg/executor/bacalhau/bacalhau.go

```diff
@@ -99,6 +99,32 @@ func (executor *BacalhauExecutor) IsAvailable() (bool, error) {
 	return true, nil
 }
 
+func (executor *BacalhauExecutor) GetMachineSpecs() ([]data.MachineSpec, error) {
+	var specs []data.MachineSpec
+	result, err := executor.bacalhauClient.getNodes()
+	if err != nil {
+		return specs, err
+	}
+
+	for _, node := range result.Nodes {
+		spec := data.MachineSpec{
+			CPU:  int(node.Info.ComputeNodeInfo.MaxCapacity.CPU) * 1000, // convert float to "mili-CPU"
```

Comment:
```suggestion
			CPU:  int(node.Info.ComputeNodeInfo.MaxCapacity.CPU) * 1000, // convert float to "milli-CPU"
```
nit: Should this be `milli`?